Makefile

```diff
@@ -0,0 +1,138 @@
+# Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+# See LICENSE.txt for license information.
+
+################################################################################
+##                             VERSION PARAMS                                 ##
+################################################################################
+
+## Docker Build Versions
+DOCKER_BUILD_IMAGE = golang:1.16.2
+DOCKER_BASE_IMAGE = alpine:3.12.4
+
+## Tool Versions
+TERRAFORM_VERSION=0.13.5
+
+################################################################################
+
+GO ?= $(shell command -v go 2> /dev/null)
+MATTERMOST_GENESIS_IMAGE ?= mattermost/genesis:test
+MACHINE = $(shell uname -m)
+GOFLAGS ?= $(GOFLAGS:)
+BUILD_TIME := $(shell date -u +%Y%m%d.%H%M%S)
+BUILD_HASH := $(shell git rev-parse HEAD)
+
+################################################################################
+
+BUILD_HASH = $(shell git rev-parse HEAD)
+LDFLAGS += -X "github.com/mattermost/genesis/model.BuildHash=$(BUILD_HASH)"
+
+# Binaries.
+TOOLS_BIN_DIR := $(abspath bin)
+GO_INSTALL = ./scripts/go_install.sh
+
+MOCKGEN_VER := v1.4.3
+MOCKGEN_BIN := mockgen
+MOCKGEN := $(TOOLS_BIN_DIR)/$(MOCKGEN_BIN)-$(MOCKGEN_VER)
+
+OUTDATED_VER := master
+OUTDATED_BIN := go-mod-outdated
+OUTDATED_GEN := $(TOOLS_BIN_DIR)/$(OUTDATED_BIN)
+
+GOVERALLS_VER := master
+GOVERALLS_BIN := goveralls
+GOVERALLS_GEN := $(TOOLS_BIN_DIR)/$(GOVERALLS_BIN)
+
+GOLINT_VER := master
+GOLINT_BIN := golint
```

Comment:
for future reference, golanci-linter is a bit better and more comprehensive https://github.com/golangci/golangci-lint (we can add it as a ticket to investigate it)

Reply:
Nice, I like it. Moved to use golangci-lint